Online-Messaging-App/src/app/home/chatbox/channel-user-list/channel-user-list.component.html

```diff
@@ -1,40 +1,44 @@
 <mat-card class="scrollMe">
-    <div class="mat-card-header">
-        <mat-card-title>
+    <div>
+        <mat-card-title class="navTitle">
             Subscribers
         </mat-card-title>
     </div>
     <mat-card>
         <mat-card-title>Online
             <hr>
         </mat-card-title>
-        <div *ngFor="let user of onlineUsers" class="onlineUsers">
-            <div (click)="goToProfile(user.username)">
-                <img
-                    class="userProfilePicture"
-                    inline="true"
-                    onerror="this.onerror=null; this.src='assets/icon-alt.PNG'"
-                    src="{{user.profileImage}}"
-                />
-                <span>{{user.username}}</span><br>
-                <span class="statusMessage"><i>{{user.statusText}}</i></span>
+        <mat-nav-list>
+            <div *ngFor="let user of onlineUsers" class="onlineUsers">
+                <mat-list-item (click)="goToProfile(user.username)" class="clickable">
+                    <img matListAvatar
+                        class="userProfilePicture"
+                        inline="true"
+                        onerror="this.onerror=null; this.src='assets/icon-alt.PNG'"
+                        src="{{user.profileImage}}"
+                    />
+                    <span matLine>{{user.username}}</span><br>
+                    <span matLine class="statusMessage"><i>{{user.statusText}}</i></span>
+                </mat-list-item>
             </div>
-        </div>
-        <br>
+            <br>
+        </mat-nav-list>
         <mat-card-title>Offline
             <hr>
         </mat-card-title>
+        <mat-nav-list>
         <div *ngFor="let user of offlineUsers" class="offlineUsers">
-            <div (click)="goToProfile(user.username)">
-                <img
+            <mat-list-item (click)="goToProfile(user.username)" class="clickable">
+                <img matListAvatar
                     class="userProfilePicture"
                     inline="true"
                     onerror="this.onerror=null; this.src='assets/icon-alt.PNG'"
                     src="{{user.profileImage}}"
                 />
-                <span>{{user.username}}</span><br>
-                <span class="statusMessage">{{user.statusText}}</span>
-            </div>
+                <span matLine>{{user.username}}</span><br>
```

Comment:
I don't know if I like the use of matLine. The content isn't long enough to justify it. I would rather you make the username bump down below the profile picture if it's too long.  This way if the web browser is full screen it is aligned with the profile picture, but as the page gets smaller it goes below the picture for the extra room. Same goes for status's. I've limited the characters on a status to 25 chararcters, so I think we should be able to see the entire status.